### PR TITLE
Streamlit App update: 

Cleaned up code and cost breakdown adjustments.

### PR DESCRIPTION
**Pull request by tiaan524**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -131,4 +132,3 @@ xerces-c=3.2.3=h0e60522_2
 xz=5.2.5=h62dcd97_1
```
Comment (tivoo) [inline requirements.txt:132]: add correct streamlit version as part of requirements

File: test proj.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import networkx as nx
```
Comment (tivoo) [inline test proj.py:1]: this file should be deleted

File: testnetwork.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
+from trenches2 import get_trench_network
```
Comment (tivoo) [inline testnetwork.py:1]: this file should also be removed

**Review by tivoo — CHANGES_REQUESTED** [review]
you need to merge into develop, not master.

**Comment by tiaan524** [comment]
Closing pull request to change base as well as update requirements.txt

**Pull request closed without merging**